finance/management/commands/xformnetfilerawdata.py

```diff
@@ -113,7 +113,7 @@ def parse_benefactor(row, verbosity=1):
 
     elif row['entity_Cd'] == 'OTH':  # Commerial benefactor or Other
         benefactor, _ = models.OtherBenefactor.objects \
-            .get_or_create(name=clean_name(row.get('tran_NamL', '')))
+            .get_or_create(name=clean_name(row.get('tran_NamL')) or '')
```

Comment:
Must have resolved the merge conflict backwards.  I'll revert it.

On Aug 14, 2016 2:24 PM, "Aaron D Borden" <notifications@github.com> wrote:

> In finance/management/commands/xformnetfilerawdata.py
> <https://github.com/caciviclab/disclosure-backend/pull/261#discussion_r74708708>
> :
>
> > @@ -113,7 +113,7 @@ def parse_benefactor(row, verbosity=1):
> >
> >      elif row['entity_Cd'] == 'OTH':  # Commerial benefactor or Other
> >          benefactor, _ = models.OtherBenefactor.objects \
> > -            .get_or_create(name=clean_name(row.get('tran_NamL', '')))
> > +            .get_or_create(name=clean_name(row.get('tran_NamL')) or '')
>
> These lines should be equivalent, get takes a the default value as the
> second argument in case the key does not exist.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/caciviclab/disclosure-backend/pull/261/files/5ff2b853270b656f80eb1b10b8618b6845c928ad#r74708708>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGcj-FsmGCRmNx8fh8UV2oQFVpqU8rqdks5qf4d_gaJpZM4Jj_Ri>
> .
>